include/sycl/algorithm/exclusive_scan.hpp

```diff
@@ -31,10 +31,13 @@
 
 #include <sycl/helpers/sycl_buffers.hpp>
 #include <sycl/helpers/sycl_namegen.hpp>
+#include <sycl/algorithm/buffer_algorithms.hpp>
 
 namespace sycl {
 namespace impl {
 
+#ifdef __USE_OLD_ALGO__
```

Comment:
I would prefer more standard macros with non `__` reserved names, such as `SYCL_PSTL_USE_OLD_ALGO` for example.